### PR TITLE
MP3 Direct Cut 20.12

### PR DESCRIPTION
**Pull request by josephsl**
## Add-on release information:

* Name: MP3 Direct Cut
* Author: Abdel, Rèmy Ruiz, Abdellah Zineddine, Jean-François Colas
* Version: 20.12
* NVDA compatibility: 2016.4 to 2019.3 (manifest states 2019.4 as last tested version but that doesn't exist)
* Add-on repo: https://github.com/abdel792/mp3DirectCut
* Release announcement: https://nvda-addons.groups.io/g/nvda-addons/message/14121

Thanks.

### CODE REVIEW

**Comment by abdel792** [comment]
Hi @josephsl,

Thanks.

Regarding compatibility, this's fixed, see the last commit in the master branch not merged yet:

* This add-on is compatible with the versions of NVDA ranging from 2016.4 until 2020.3.

File: get.php [hunk 1]
```diff
@@ -97,7 +97,7 @@
 	"rccp" => "https://github.com/tuukkao/nvda-reviewCursorCopier/releases/download/1.2/reviewCursorCopier-1.2.nvda-addon",
 	"rccp-dev" => "https://github.com/tuukkao/nvda-reviewCursorCopier/releases/download/1.2/reviewCursorCopier-1.2.nvda-addon",
 	"rf" => "https://github.com/nvdaes/readFeeds/releases/download/9.0/readFeeds-9.0.nvda-addon",
-	"rf-dev" => "https://github.com/nvdaes/readFeeds/releases/download/9.0-dev/readFeeds-9.0-dev.nvda-addon",
+	"rf-dev" => "https://github.com/nvdaes/readFeeds/releases/download/10.11-dev/readFeeds-10.11-dev.nvda-addon",
 	"rf-o" => "https://github.com/nvdaes/readFeeds/releases/download/4.5/readFeeds-4.5.nvda-addon",
```
Comment (feerrenrut) [inline get.php:101]: Should this be here? The PR description only mentions mp3dc.

**Comment by feerrenrut** [comment]
@josephsl I'll hold off on this. I'm waiting for clarification about the compatibility metadata. Does a new release need be made to incorporate the commit that @abdel792 refers to?

**Comment by josephsl** [comment]
Hi, ideally a new commit would be helpful, but since 2019.3 covers “2019.4”, I would say it doesn’t matter. Thanks.

**Comment by josephsl** [comment]
Hi, oops, will send a forced commit. Thanks.

**Comment by abdel792** [comment]
Hi @josephsl & @feerrenrut,

Sorry, I forgot to update the buildVars.py file, I just fixed it in the last commit.

In my previous post, I was only talking about the compatibility information listed in readme.md.

Thanks.

**Comment by nvdaes** [comment]
Hello: as a reviewer, I can confirm that buildvars.py is updated. Abdel and me (and other authors) are subscribed to a mailing list created by me and we received automatic emails with notification for our add-on commits and tags. I created a Google script for this (though hope the new system can include something similar used for other add-ons and authors if accepted).
So I've received a message with the following link:
https://github.com/abdel792/mp3DirectCut/compare/b75109d25c24...4fc22515c401

Reviewing this I can confirm the mentioned change.
Thanks

**Comment by abdel792** [comment]
Hi dear friends,

I don't understand why the abdel792/mp3directcut master branch has not yet been merged into nvdaaddons/mp3directcut master branch yet?

I cannot currently do my PR on nvaccess / addonfiles on get.php file until this merge has been done.

Thanks.

**Comment by nvdaes** [comment]
Dear Abdel, sorry, I'm lost with add-ons management now in this transition. But I see this line reviewing this pr:

+	"mp3dc" => "https://github.com/abdel792/mp3DirectCut/releases/download/v20.12/mp3DirectCut-20

So you don't need that we merge this into nvdaaddons org. Anyway, I think that due to this transition and the different ways we release add-ons we can be confused, so excuse me if I'm missing something.
As an example, yesterday someone sayd that there is another addon maintained by you for which addonUpdater was not offering the correct version to update, and other person explained that the version installed by the first one is not official, so the problem was not in addonUpdater.
Not sure...

**Comment by abdel792** [comment]
Hi @feerrenrut,

Please could you correct line 68 in get.php file as follows?

    -	"mp3dc" => "https://github.com/abdel792/mp3DirectCut/releases/download/v19.12/mp3DirectCut-19.12.nvda-addon",

    +	"mp3dc" => "https://github.com/abdel792/mp3DirectCut/releases/download/v20.12/mp3DirectCut-20.12.nvda-addon",

This will update the download link of stable version.

Thanks in advance.

**Comment by nvdaes** [comment]
Sorry, this was just an example, but the thread was posted on my
spanish mailing list.


2020-12-11 9:01 GMT+01:00, abdel792 <notifications@github.com>:

**Comment by abdel792** [comment]
Thanks @feerrenrut.

**Pull request merged**